src/components/common/PagePlaceholder/index.tsx

```diff
@@ -3,17 +3,17 @@ import { Typography } from '@mui/material'
 import css from './styles.module.css'
 
 type PagePlaceholderProps = {
-  imageUrl: string
+  img: ReactNode
   text: string
   children?: ReactNode
 }
 
-const PagePlaceholder = ({ imageUrl, text, children }: PagePlaceholderProps): ReactElement => {
+const PagePlaceholder = ({ img, text, children }: PagePlaceholderProps): ReactElement => {
   return (
     <div className={css.container}>
-      <img src={imageUrl} alt="background" />
+      {img}
 
-      <Typography variant="h3" m={3}>
+      <Typography variant="body1" color="primary.light">
```

Comment:
I think in the designs we always have 16pxs margin-top:

```suggestion
      <Typography variant="body1" mt={2} color="primary.light">
```


![localhost_3000_](https://user-images.githubusercontent.com/2670790/191725383-0f28f61c-f560-49a7-a379-a2d3dc0a4b06.png)